cibuildwheel/__main__.py

```diff
@@ -154,22 +164,38 @@ def main():
         build_selector=build_selector,
         repair_command=repair_command,
         environment=environment,
+        dependency_constraints=dependency_constraints,
     )
 
     if platform == 'linux':
         manylinux_x86_64_image = os.environ.get('CIBW_MANYLINUX_X86_64_IMAGE', 'manylinux2010')
         manylinux_i686_image = os.environ.get('CIBW_MANYLINUX_I686_IMAGE', 'manylinux2010')
 
-        default_manylinux_images_x86_64 = {'manylinux1': 'quay.io/pypa/manylinux1_x86_64',
-                                           'manylinux2010': 'quay.io/pypa/manylinux2010_x86_64',
-                                           'manylinux2014': 'quay.io/pypa/manylinux2014_x86_64'}
-        default_manylinux_images_i686 = {'manylinux1': 'quay.io/pypa/manylinux1_i686',
-                                         'manylinux2010': 'quay.io/pypa/manylinux2010_i686',
-                                         'manylinux2014': 'quay.io/pypa/manylinux2014_i686'}
+        pinned_docker_images_file = os.path.join(
+            os.path.dirname(__file__), 'resources', 'pinned_docker_images.cfg'
+        )
+        pinned_docker_images = ConfigParser()
+        pinned_docker_images.read(pinned_docker_images_file)
+
+        # pinned_docker_images looks like a dict of dicts, e.g.
+        # { 'manylinux1': {'x86_64': '...', 'i686': '...'},
+        #   'manylinux2010': {'x86_64': '...', 'i686': '...'},
+        #   'manylinux2014': {'x86_64': '...', 'i686': '...'} }
+
+        manylinux_images = {}
+
+        if manylinux_x86_64_image in pinned_docker_images:
```

Comment:
Yes, it's a separate setting. I've added a note to the docs about how to use the latest image https://github.com/joerick/cibuildwheel/pull/256/files#diff-76c9849489bb8e65c3d8b38221c9e876R263

`CIBW_MANYLINUX_X86_64_IMAGE =manylinux:latest` isn't worth the special-casing and extra documentation, in my opinion

Reply:
OK, yes, looking back at this, that's true. I should first check if it's already there, but otherwise, a summary of what is/can be pinned and how might be nice?